### PR TITLE
Simplify delete modal interface

### PR DESCRIPTION
**Pull request by eatonphil**

### CODE REVIEW

File: src/linodes/layouts/IndexPage.js [hunk 1]
```diff
@@ -89,6 +89,10 @@ export class IndexPage extends Component {
   deleteLinodes(linodesToBeRemoved) {
     const { dispatch, selectedMap, linodes } = this.props;
 
+    const selectedLinodes = Object.values(linodes.linodes)
```
Comment (mparke) [inline src/linodes/layouts/IndexPage.js]: double check that the param of this function is, might have everything we need without needing to mess with the selectedMap

**Comment by eatonphil** [comment]
@mparke done, good call

**Comment by abemassry** [comment]
clicked Delete button next to nodebalancer
image

**Comment by abemassry** [comment]
image

should say Zones, or Domains may get fixed in a future update

**Comment by eatonphil** [comment]
@abemassry NodeBalancers aren't handled correctly at all. But I've got a fix for the last one coming in a second.

**Comment by eatonphil** [comment]
thank you!

**Pull request merged**